Documentation/workflow/EventPipeDeveloperGuide.md

```diff
@@ -0,0 +1,105 @@
+
+# Event Pipe Developer Guide
+
+The goal of this guide is to give a good introduction for new contributors who would like to develop Event Pipe.
+
+## What is Event Pipe?
+
+It's new, cross-platform mechanism for fast event tracing in .NET Core. [Here](https://github.com/dotnet/designs/blob/master/accepted/cross-platform-performance-monitoring.md) you can read more about its design. 
```

Comment:
It'd be useful to compare it's function to LTTng and ETW, as those are what people will likely be familiar with